_component_design/text-fields.md

```diff
@@ -132,17 +103,16 @@ pagelink: https://rackerlabs.github.io/helix-ui/components/text-inputs/
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Required fields
+### Style
```

Comment:
need to fix some redline issues in the visual. look for /style in the png folder.